src/main/java/com/iotiq/application/domain/Seller.java

```diff
@@ -30,6 +33,12 @@ public class Seller extends AbstractPersistable<UUID> {
 
     private boolean isActive;
 
+    private List<Skill> skills;
```

Comment:
Will the elements need to be distinct? If so, a set would fit better imo.

Reply:
Also, how does hibernate handle this in the database? Does it get saved as a blob or a json list etc.?

Reply:
I used    
@ElementCollection(targetClass = Skill.class)
 @Enumerated(EnumType.STRING)

Reply:
I will make arrangements to check duplicate for all enum lists

Reply:
> Will the elements need to be distinct? If so, a set would fit better imo.
@oguzIotiq how would you handle this?

My thinking was to use a set so that the data structure handles the duplicates implicitly. Would it prevent us from doing anything?